src/test/resources/docker/docker-compose.yml

```diff
@@ -284,7 +284,7 @@ services:
 
   mssql-2019:
     container_name: sql-server-db-2019
-    image: mcr.microsoft.com/mssql/server:2019-CU27-ubuntu-20.04
+    image: mcr.microsoft.com/mssql/server:2019-latest
```

Comment:
can we rollback this to `-CU27-ubuntu-20.04` ? This is not related to the runners update changes.